utils/fleaflicker.js

```diff
@@ -0,0 +1,30 @@
+// some functions to interact with the fleaflicker fantasy API
+// https://www.fleaflicker.com/api-docs/index.html
+
+import fetch from "node-fetch";
+
+const base_url = "https://www.fleaflicker.com/api";
+
+export async function fetchLeagueActivity({ sport = "MLB", league_id = "24736" }) {
+  const post_data = { sport: sport, league_id: league_id };
+  const response = await fetch(`${base_url}/FetchLeagueActivity`, {
+    method: "POST",
```

Comment:
Should these be GETs?

Reply:
The api was fine with posts when I was testing with node, but yeah the docs do specify get, good call. Guess I gotta send a query string with the data instead of a body